tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -443,11 +456,18 @@ private void PostIteration(ScenarioExecutionResult scenarioExecutionResult)
                         continue;
                     }
 
-                    match = Regex.Match(line, @"^Steadystate average response time: (\d+)ms$");
+                    match = Regex.Match(line, @"^Steadystate average response time: (\d+\.?\d*)ms$");
                     if (match.Success && match.Groups.Count == 2)
                     {
                         steadyStateAverageTime = Convert.ToDouble(match.Groups[1].Value);
-                        break;
+                        continue;
+                    }
+
+                    match = Regex.Match(line, @"^Steadystate median response time: (\d+\.?\d*)ms$");
+                    if (match.Success && match.Groups.Count == 2)
+                    {
+                        steadyStateMedianTime = Convert.ToDouble(match.Groups[1].Value);
+                        continue;
```

Comment:
>continue [](start = 24, length = 8)

Should we break here?

Reply:
No, because the app now emits 5 pairs of average/median results, and we want the last.